src/AboutMe/aboutMe.js

```diff
@@ -1,13 +1,70 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import manSit from "./manOnTable.svg";
+import "./aboutMe.css";
+import Button from "react-bootstrap/Button";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { FaHtml5, FaCss3Alt, FaPython, FaJava } from "react-icons/fa";
+import { SiCplusplus } from "react-icons/si";
 
 class AboutMe extends Component {
-    render() {
-        return (
-            <div className="aboutMe">
-                <h1> title</h1>
+  render() {
+    return (
+      <div className="greet-main" id="greeting">
+        <div className="greeting-main">
+          <div className="greeting-text-div">
+            <div>
```

Comment:
not sure if div necessary? I see the div above it with className "greeting-text-div" also encloses the same elements.